ci/util.py

```diff
@@ -332,6 +332,11 @@ def merge_dicts(base: dict, other: dict, list_semantics='merge'):
     return merger.merge(deepcopy(base), deepcopy(other))
 
 
+def sane_env_var_name(name: str):
```

Comment:
leave a copy of the moved function where it was + add a @deprecated annotation to it (remove after relasing a.s.o.)